File: Desenvolvimento/server/src/ApplicationCore/Entities/FluxoItem.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class FluxoItem : IBaseEntity<long>
    {
        public long Id { get; set; }

        public long FluxoId { get; set; }
        public virtual Fluxo Fluxo { get; set; }

        public long SituacaoAtualId { get; set; }
        public virtual FluxoSituacao SituacaoAtual { get; set; }

        public long AcaoId { get; set; }
        public virtual FluxoAcao Acao { get; set; }

        public long ProximaSituacaoId { get; set; }
        public virtual FluxoSituacao ProximaSituacao { get; set; }
        
        public virtual ICollection<FluxoItemSetor> FluxoItemSetores { get; set; } = new List<FluxoItemSetor>();
        public virtual ICollection<FluxoItemTipoAnexo> FluxoItemTiposAnexo { get; set; } = new List<FluxoItemTipoAnexo>();
        public virtual ICollection<FluxoItemChecklist> FluxoItemChecklists { get; set; } = new List<FluxoItemChecklist>();
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/fluxo-item.ts
import { Fluxo } from './fluxo';
import { FluxoSituacao } from './fluxo-situacao';
import { FluxoAcao } from './fluxo-acao';
import { FluxoItemCheckList } from './fluxo-item-checklist';
import { FluxoItemSetor } from './fluxo-item-setor';
import { FluxoItemAnexo } from './fluxo-item-anexo';
import { BaseModel } from '../base-model';

export class FluxoItem extends BaseModel {
    constructor(
        public id: number,
        public fluxoId?: number,
        public fluxo?: Fluxo,
        public situacaoAtualId?: number,
        public situacaoAtual?: FluxoSituacao,
        public acaoId?: number,
        public acao?: FluxoAcao,
        public proximaSituacaoId?: number,
        public proximaSituacao?: FluxoSituacao,
        public fluxoItemSetores?: Array<FluxoItemSetor>,
        public fluxoItemTiposAnexo?: Array<FluxoItemAnexo>,
        public fluxoItemChecklists?: Array<FluxoItemCheckList>
    ) { super(); }
}
